wadsrc/static/zscript/shared/dynlights.txt

```diff
@@ -28,7 +28,9 @@ class DynamicLight : Actor native
 		ColorFlickerLight, 
 		RandomColorFlickerLight
 	};
-	
+
```

Comment:
If there were two empty lines then leave them where they are and add your code below.
The less useless noise PR has, the faster it will be accepted.

Reply:
There weren't really two lines above.